src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.ts

```diff
@@ -187,6 +187,16 @@ export class RollCallComponent implements OnChanges {
     }
   }
 
+  goToFirst() {
+    this.goToParticipantWithIndex(0);
+  }
+
+  goToLast() {
+    // No need for `this.children.length - 1`
+    // because there is one more page than there are children
```

Comment:
Lets make this comment a little less confusion and say that we just finalize the roll call instead of jumping to the last participant.